.ci/jenkins_job_build.sh

```diff
@@ -45,12 +45,12 @@ export PATH=${GOPATH}/bin:/usr/local/go/bin:/usr/sbin:/sbin:${PATH}
 kata_repo_dir="${GOPATH}/src/${kata_repo}"
 tests_repo_dir="${GOPATH}/src/${tests_repo}"
 
-arch=$("${tests_repo_dir}/.ci/kata-arch.sh")
-
 # Get the tests repository
 mkdir -p $(dirname "${tests_repo_dir}")
 [ -d "${tests_repo_dir}" ] || git clone "https://${tests_repo}.git" "${tests_repo_dir}"
 
+arch=$("${tests_repo_dir}/.ci/kata-arch.sh")
+
```

Comment:
as per comment on the closed PR - looks fine, as we are already pulling the repo, this is just a change in order.

@Pennyzct - you didn't need to close the other PR and open a new one - you could have done some git `rebase -i` and rebase magic I think to have fixed it and then force pushed a new version :-)

Reply:
yep, you caught me, I `git checkout` this branch based on my another development, not the `HEAD`. 😭. And I quickly cleaned up the evidence in upstream. 
I must do some dig up on `git rebase` command. This is how my little brain could think of, `git format-patch` to save my commit, delete this branch and then re-create it based on `HEAD`, and finally `git am`, 🤢🤮